CHANGELOG.md

```diff
@@ -43,11 +44,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime 1
 
-- Nothing.
+- Option to use deterministic Incinerator Drone RNG for fairer racing
 
 #### Patcher Changes
 
-- Option to use deterministic Incinerator Drone RNG for fairer racing
+- Added: QoL Game Breaking - Reserach Lab Aether Pirate now guaranteed to jump through glass when doing room backwards
+- Fixed: Players could unmorph in Magmoor Workstation where they should not be able to
+- Fixed: Abuse of QoL Game Breaking in Central Dynamo to skip the maze/drone
+- Fixed: Exclude Phazon Elite Item from QoL Pickup Scans
+- Fixed: Wavesun when playing with shuffled item positions
+- Fixed: Main Plaza etank ledge door shield was slightly misaligned
+- Fixed: Cannon remaining holstered after grapple when shuffling combat visor
+- Fixed: Cannon remaining holstered after a specific type of R-Jump when shuffling combat visor
+- Fixed: Unmorphing now returns you to your previous visor instead of default visor when shuffling combat visor for quality of life purposes
+- Changed: Spring Ball now has 40 frames of cooldown
```

Comment:
4.0 doesn't have spring ball, and I don't see any entries about adding it. This should be changed into a "Added Spring Ball" entry